src/scss/constants.scss

```diff
@@ -304,6 +306,12 @@ $z-layers: (
     background-color: $black;
   }
 
+  &::-webkit-scrollbar-thumb:hover {
+    background-color: rgba($white, 0.5);
+  }
+}
+
+@mixin gray-scrollbar {
```

Comment:
There shouldn't be two different scrollbars though. If it needs tweaking for one specific component, it can be probably targeted from it, like

```scss
.ParentComponent {
  &::-webkit-scrollbar...
}
```

Reply:
fixed in that PR https://github.com/sparkletown/sparkle/pull/2387